src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -72,7 +72,7 @@
     background-color: var(--token-color-palette-neutral-50);
     border-radius: 5px;
     border: 1px solid rgba(101, 106, 118, 0.2);
-    font-size: 13px;
+    font-size: 0.8125rem;
```

Comment:
Bit of a random tangent, but figured might as well use an equivalent `rem` value here to match practices throughout the rest of the project.